assets/css/nav/_left_nav.scss

```diff
@@ -0,0 +1,82 @@
+.m-left-nav {
+  width: calc(11.5rem + 1px);
+  display: flex;
```

Comment:
We don't have a particular rule about ordering either rules or the attributes within them, but it would probably be good to adopt a convention of some sort.